app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -89,8 +93,6 @@ def sync_workers
         end
       end
 
-      modify_apache_ports(ports_hash, self::PROTOCOL) if MiqEnvironment::Command.supports_apache?
-
```

Comment:
We no longer modify the configuration after adding/removing Ui/Web Service/Web Socket workers...